packages/calcite-components/src/components/input-text/input-text.scss

```diff
@@ -1,4 +1,43 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-input-text-corner-radius: defines the border radius of the component.
+ * @prop --calcite-input-text-text-color: defines the text color of the component.
```

Comment:
Similar to `input` - add `-input-text-shadow` that defaults to "none" (although, in this case, it is confusing to call it -input-text-shadow, lol)